app/src/controllers/filter/sagas.js

```diff
@@ -174,6 +177,18 @@ function* saveNewFilter({ payload: filter }) {
   yield put(changeActiveFilterAction(newId));
 }
 
+function* parseQueryToFilterEntity() {
+  const filterConditions = yield select(filterFromQuerySelector);
+  const projectId = yield select(projectIdSelector);
+
+  if (filterConditions) {
+    yield put(createFilterAction(filterConditions));
+    yield put(redirect({ type: LAUNCHES_PAGE, payload: { projectId } }));
```

Comment:
This leads to the double launches request to the server.
Can we add a parameter to the `createFilterAction` to dispatch the next page action with redirect and remove this line?